docs/DeveloperGuide.md

```diff
@@ -824,18 +824,114 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
 - `add n/ e/brendanl@gmail.com p/96734294 a/Blk 653C Jurong West Street 61 Singapore 643653 t/PRUGrowth`.
 - `add n/Brendan Lim n/Lim Wei Sheng Brendan e/brendanl@gmail.com p/96734294 a/Blk 653C Jurong West Street 61 Singapore 643653 t/PRUGrowth`.
 
+### Viewing all persons
+#### Viewing all persons successfully
+**Example Test Case:** `list`.
+
+**Expected Result:**
+1. *Person List* shows all persons in FINDvisor.
+2. *Command Result Box* outputs a successful execution message.
+3. *Command Box* is cleared.
+
+**Alternative Test Cases:**
+- `list 1`.
+- `list a`.
+
+Alternative test cases should have similar expected results.
+
+### Editing a person
+#### Editing a person successfully
+**Prerequisites:**
+1. List all persons using the `list` command. At least 2 persons in the list.
+2. No person in FINDvisor has the mobile phone number `96734294`.
+
+**Example Test Case:** `edit 1 p/96734294 e/jameslee@example.com a/123, Clementi Rd, 1234665 t/PRUGrowth`.
+
+**Expected Result:**
+1. *Person List* shows all persons in FINDvisor. The respective person's fields will be overwritten with the specified values supplied in the command.
+2. *Command Result Box* outputs a successful execution message with the edited person's details.
+3. *Command Box* is cleared.
+
+**Alternative Test Cases:**
+1. `edit 2 n/James Lee e/jameslee@example.com a/123, Clementi Rd, 1234665 t/PRUGrowth`.
+2. `edit 1 n/John Tan`
+
+#### Editing a person who has a meeting scheduled today successfully
+1. List all persons using the `list` command. At least 2 persons is displayed in the list.
+3. The first person in the list is also has a meeting scheduled today.
+
+**Example Test Case:** `edit 1 n/James Lee e/jameslee@example.com a/123, Clementi Rd, 1234665`.
+
+**Expected Result:**
+1. *Today's Meeting List* shows updated person's fields (name and phone number only) based on the specified values supplied in the command.
+2. *Person List* shows all persons in FINDvisor. The respective person's fields will be overwritten with the specified values supplied in the command.
+3. *Command Result Box* outputs a successful execution message with the edited person's details.
+4. *Command Box* is cleared.
+
+#### Invalid value or command format used for editing a person
+**Prerequisites:**
+1. List all persons using the `list` command. Multiple persons are in the list (Less than 100).
```

Comment:
```suggestion
1. List all persons using the `list` command. Multiple persons are in the list (less than 100).
```
Just a nit but I think we should use lower case for `less`?

Should we also update all cases to state `less than 100` instead of `100000`?